src/MultipartStreamBuilder.php

```diff
@@ -198,4 +203,28 @@ public function setBoundary($boundary)
 
         return $this;
     }
+
+    /**
+     * @return MimetypeHelper
+     */
+    private function getMimetypeHelper()
+    {
+        if ($this->mimetypeHelper === null) {
+            $this->mimetypeHelper = new ApacheMimetypeHelper();
+        }
+
+        return $this->mimetypeHelper;
+    }
+
+    /**
```

Comment:
i would add a doc here that explains this is to overwrite the default mimetype helper if one needs custom behaviour.